runtime/compiler/control/rossa.cpp

```diff
@@ -1868,13 +1868,24 @@ aboutToBootstrap(J9JavaVM * javaVM, J9JITConfig * jitConfig)
 #if defined(J9VM_OPT_SHARED_CLASSES)
    if (isSharedAOT)
       {
-      /* If AOT Shared Classes is turned ON, perform compatibility checks for AOT Shared Classes */
-      if (0) //!validateSharedClassAOTHeader(javaVM, curThread, compInfo))
+      /* If AOT Shared Classes is turned ON, perform compatibility checks for AOT Shared Classes
+       *
+       * This check has to be done after latePostProcessJIT so that all the necessary JIT options
+       * can be set
+       */
+      TR_J9SharedCacheVM *fe = (TR_J9SharedCacheVM *) TR_J9VMBase::get(jitConfig, curThread, TR_J9VMBase::AOT_VM);
```

Comment:
Do we actually need a TR_J9SharedCacheVM frontend? As far as I can tell we only need the frontend to ask if it asynchronous compilation is disabled.

Reply:
Probably not, I mean given that we're calling code to validate the AOT Header, the SCC must be enabled.

Reply:
Addressed in 0233e1d2ebdd5ba3beae46cb7d6b7f0a1c2f8b06